python/risk_meters/historical_vuln_count.py

```diff
@@ -0,0 +1,187 @@
+    for (vc_date, ra_date) in zip(vuln_count_data, risk_accepted_data):
+        if vc_date != ra_date:
+            print(f"Dates don't match: {vc_date} : {ra_date}")
+            sys.exit(1)
+
+        # Add today or the first of the month to the table.
+        if one_shot or vc_date[-2:] == "01":
+            vc_high = int(vuln_count_data[vc_date]['high'])
+            vc_medium = int(vuln_count_data[vc_date]['medium'])
+            vc_low = int(vuln_count_data[vc_date]['low'])
+            ar_high = int(risk_accepted_data[ra_date]['high'])
+            ar_medium = int(risk_accepted_data[ra_date]['medium'])
+            ar_low = int(risk_accepted_data[ra_date]['low'])
+   
+            vuln_count_tbl.add_row([vc_date, vc_high, ar_high, vc_medium, ar_medium, vc_low, ar_low])
+        
+    print(vuln_count_tbl)
+    print("")
+
+def print_help(program):
+    print(f"Prints out the historical risk meter vulnerability counts for a year.")
+    print(f"{program} <risk meter name> [today]")
+    print(f"   <risk meter name> - name of the risk meter")
+    print(f"   [today] - if present will only print the vulnerability counts for today")
+    print("")
+
+if __name__ == "__main__":
+    # Set the one_shot flag to False.
+    one_shot = False
+    check_risk_meter_name = True
+
+    # Look for the risk meter name.
+    if len(sys.argv) > 1:
+        risk_meter_name = sys.argv[1]
+    else:
+        print("Requires risk meter name.")
+        print_help(sys.argv[0])
+        sys.exit(1)
+
+    # Look for the "today" keyword which tells this program to do only
+    # today and not monthly for a year.
+    if len(sys.argv) > 2:
+        if sys.argv[2] == "today":
+            one_shot = True
+        else:
+            print("Currently only 'today' is supported.")
+            print_help(sys.argv[0])
+            sys.exit(1)
+
+    # Check for arguments that could show if run as a subprocess.
+    # If the paramters are invalid, ignore them and proceed.
+    # If the parameters are valid, set the check_risk_meter_name to False.  This
+    # reduces the number of API invokes.
+    if len(sys.argv) > 3 and len(sys.argv) < 7:
+        try:
+            risk_meter_id = int(sys.argv[3])
+            risk_meter_score = int(sys.argv[4])
+            risk_accepted_score = int(sys.argv[5])
+            check_risk_meter_name = False
+        except ValueError:
+            print(f"Invalid integer argument: {sys.argv[3]}, {sys.argv[4]}, {sys.argv[5]}")
+            check_risk_meter_name = True
+
+    # Only print the title if checking the risk meter name impling that this is not a subprocess.
+    if check_risk_meter_name:
+        print("")
+        print("Historical Risk Meter Vulnerability Counts")
+        print("")
+
+    # Obtain the Kenna Security API key from an environment variable.
+    api_key = os.getenv('KENNA_API_KEY')
+    if api_key is None:
+        print("API key is non-existent")
```

Comment:
I'd make this more user-centric like, "In order to run this script, please assign your Kenna Risk Token to the KENNA_API_KEY environment variable."

Reply:
Done.